types/lhr/audit-details.d.ts

```diff
@@ -117,6 +117,7 @@ declare module Details {
   interface TableSubItems {
     type: 'subitems';
     items: TableItem[];
+    label?: string | IcuMessage;
```

Comment:
The duplication of the label for every group of subItems is unfortunate. What about defining this at a higher level (like a property of the table itself) and we can add overriding at the `TableSubItems` level if/when we need it?

Reply:
It seems likely this will come up for inspector-issues one day, but yeah we could move this up to the Table

<img width="397" alt="image" src="https://user-images.githubusercontent.com/4071474/163637964-689e5899-fa61-4e26-832d-99d138075445.png">


but would you prefer the extra complexity to (probably, one day) support inheritance vs the extra few bytes in the LHR?